packages/manager/modules/anthos/package.json

```diff
@@ -20,7 +20,9 @@
     "@ovh-ux/ui-kit": "^4.4.3",
     "@uirouter/angularjs": "^1.0.23",
     "angular": "^1.7.5",
+    "angular-chart.js": "^1.1.1",
```

Comment:
New `peerDependencies` mean that it must be added as `dependencies` on the parent (standalone) application.
see: packages/manager/apps/anthos/package.json

Reply:
Thanks! This is done